service/resource/legacy/resource.go

```diff
@@ -986,8 +986,12 @@ func (s *Resource) processDelete(cluster awstpr.CustomObject) error {
 		return microerror.Maskf(executionFailedError, fmt.Sprintf("cluster spec is invalid: '%#v'", err))
 	}
 
-	if err := s.deleteClusterNamespace(cluster.Spec.Cluster); err != nil {
-		s.logger.Log("error", "could not delete cluster namespace:", err)
+	// For new clusters using Cloud Formation there is an OperatorKit resource
+	// for the k8s namespace.
+	if !key.UseCloudFormation(cluster) {
+		if err := s.deleteClusterNamespace(cluster.Spec.Cluster); err != nil {
+			s.logger.Log("error", "could not delete cluster namespace:", err)
+		}
```

Comment:
This is the same disabling logic. I'm fine with switching to resource routing if that is possible.